tools/esbuild-make

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# This file is part of Cockpit.
+#
+# Copyright (C) 2023 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import argparse
+import os
+import subprocess
+
+parser = argparse.ArgumentParser(description='Run esbuild')
+parser.add_argument('-l', '--no-lint', action='store_true', help='Skip linting')
+parser.add_argument('-r', '--rsync', metavar='HOST', default='', help='rsync esbuild output to ssh target after build')
+parser.add_argument('-w', '--watch', action='store_true', help='Enable watch mode')
+parser.add_argument('prefix', nargs='?', metavar='DIRECTORY', help='The directory to build (e.g base1, shell, ....)')
+args = parser.parse_args()
+
+if args.watch:
+    os.environ["ESBUILD_WATCH"] = "1"
+if args.no_lint:
+    os.environ["LINT"] = "0"
+if args.rsync:
+    os.environ["RSYNC"] = args.rsync
+if args.prefix:
+    os.environ["ONLYDIR"] = args.prefix
+
+subprocess.check_call(["node", "build.js"])
```

Comment:
It seems to me that we should just add the arg parsing to build.js directly, and then our entry point just becomes `./build.js`. Let's clean up all these wrappers. Again, this can be a follow-up.

Reply:
I am fine with this, just thought that people have gotten used to the scripts being in ./tools/.  Plus there are still one there, like the jumpstart.